### PR TITLE
Create a reusable Layout component for page layout

### LINKED ISSUE #252: Thoughts - Fix layout duplication on ThoughtsPage and MainPage
Create a new Layout component to encompass the TopNavbar and TagsSidebar to stop duplication across pages.

https://github.com/wasp-lang/wasp/blob/master/examples/thoughts/ext/MainPage.js#L32
https://github.com/wasp-lang/wasp/blob/master/examples/thoughts/ext/ThoughtsPage.js#L21

### PR DESCRIPTION
**Pull request by cursorial**
# Description

In order to remove duplication for the TopNavbar and the TagsSidebar on
these pages we need a base Layout component. This PR eliminates the
duplication we were seeing before.

Fixes #252 

## Type of change

Please select the option(s) that is more relevant.

- [x] Code cleanup
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: examples/thoughts/ext/Layout.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.base-page {
```
Comment (Martinsos) [inline examples/thoughts/ext/Layout.css:1]: You took `main-page` and `main-container` names and turned them into `base-page` and `base-container`.
I wonder if this is a good opportunity to find them even better names?
Maybe `layout-container`/`layout-root` and `layout-content`?

**Review by Martinsos — APPROVED** [review]
Great! I think all is clear here, this is much nicer now. I left just one comment regarding css class names, take a look.

**Comment by Martinsos** [comment]
@cursorial how is this going, do you need anything from me to proceed?

File: examples/thoughts/ext/Layout.css (hunk 1 above)
Comment (cursorial) [reply examples/thoughts/ext/Layout.css]: I like `layout-root` and `layout-content`. Changed these now.

**Comment by cursorial** [comment]
> @cursorial how is this going, do you need anything from me to proceed?

All good, just haven't made time to look at this recently. Updated now.

**Comment by Martinsos** [comment]
Awesome, merged!

**Pull request merged**